src/Api/Controller/ListDiscussionsController.php

```diff
@@ -110,9 +110,13 @@ protected function data(ServerRequestInterface $request, Document $document)
 
         Discussion::setStateUser($actor);
 
+        if (in_array('mostRelevantPost.user', $include)) {
```

Comment:
I'd add a comment saying this is done for eager loading

Reply:
Do we want to explain why we need to eager load it or just state that we're eager loading it ?

Observation: this is currently only necessary because the suspend extension's UserPolicy calls `isAdmin` on the `$user` object which loads all groups, in which case we should be eager loading the relation in the extension and not here, but I'm seeing in https://github.com/flarum/core/pull/2620/files#diff-5f5aff0db193a1b7b5bdff796982aefe84572973059c31c65f33233e7fafd597R34-R36 that we might end up doing it in core as well, so it will belong here after all (if that pr isn't changed).

Reply:
Should the comment reflect that? Suggestion:

    // Eager load groups for use in the policies (isAdmin check)